package.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
should we add `yarn lint` as pre-push hook?

Reply:
happy to add, saves pushing broken stuff hopefully 😅